ols/constants.py

```diff
@@ -51,9 +51,18 @@ class QueryValidationMethod(StrEnum):
 GPT35_TURBO_1106 = "gpt-3.5-turbo-1106"
 GPT35_TURBO = "gpt-3.5-turbo"
 
-# Model configs
+
+# Token related constants
 DEFAULT_CONTEXT_WINDOW_SIZE = 8000
 DEFAULT_RESPONSE_TOKEN_LIMIT = 500
+MINIMUM_CONTEXT_TOKEN_LIMIT = 1
+
+DEFAULT_TOKENIZER_MODEL = "cl100k_base"
+
+
+# RAG related constants
+RAG_CONTENT_LIMIT = 1
+RAG_SIMILARITY_CUTOFF = 0.5  # Lower is better
```

Comment:
please add more comments explaining what this constant does/how it affects results when the value is changed